packages/build/tests/core/tests.js

```diff
@@ -431,6 +431,26 @@ test('Shows notice about modules with dynamic imports and suggests the usage of
   await runFixture(t, 'esbuild_errors_2')
 })
 
+const generate250mbFile = (path) => {
```

Comment:
Would the following be simpler?

```js
await pWriteFile(path, `module.exports = '${'a'.repeat(50000000)}'`)
```

With `pWriteFile` being `util.promisify(fs.writeFile)`.

On my machine, it is also ~45 times much faster: ~200ms vs ~9000ms

Reply:
repeating 'a' that much won't make the ZIP file grow big enough, because a static 'a' can be compressed really well. But I like the idea of just using `writeFile` as opposed to write streams :D somehow I thought that'd be too slow for a 250mb file. 879b95b

Reply:
Good point about the zip compression!